src/pycram/ontology.py

```diff
@@ -0,0 +1,507 @@
+import inspect
+from pathlib import Path
+from typing import Optional, List, Type, Callable
+
+from owlready2 import *
+
+from pycram.enums import ObjectType
+from pycram.helper import Singleton
+from pycram.designator import DesignatorDescription, ObjectDesignatorDescription
+
+SOMA_HOME_ONTOLOGY = "http://www.ease-crc.org/ont/SOMA-HOME.owl"
+SOMA_ONTOLOGY = "http://www.ease-crc.org/ont/SOMA.owl"
+
+class OntologyManager(object, metaclass=Singleton):
+    """
+    Singleton class as the adapter accessing data of an OWL ontology, largely based on owlready2.
+    """
+
+    """
+    The ontology instance as the result of an ontology loading operation
```

Comment:
i am fairly confident that docstrings for class variables should be below the variable declaration and not above. Please check that this is rendered as you intended by building the doc.